ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp

```diff
@@ -151,7 +151,7 @@ void Geotriggers::createGeotriggerMonitor(ServiceFeatureTable* serviceFeatureTab
 void Geotriggers::handleGeotriggerNotification(GeotriggerNotificationInfo* geotriggerNotificationInfo)
 {
   // Create a QScopedPointer so GeotriggerNotificationInfo does not persist outside this scope
-  QScopedPointer<GeotriggerNotificationInfo> scopedNotification(geotriggerNotificationInfo);
+  std::unique_ptr<GeotriggerNotificationInfo> scopedNotification(geotriggerNotificationInfo);
```

Comment:
Probably no reason to continue to use `QScopedPointer`.

Reply:
no, it's not deprecated. And until we don't want to move the pointer, it's not bad to use `QScopedPointer`. I can create an issue in sample repo.